psiminer-core/src/main/kotlin/psi/transformations/excludenode/ExcludePhpSymbolsTransformation.kt

```diff
@@ -0,0 +1,13 @@
+package psi.transformations.excludenode
+
+import com.intellij.psi.PsiElement
+import com.intellij.psi.util.elementType
+import psi.transformations.PhpTreeTransformation
+
+class ExcludePhpSymbolsTransformation : PhpTreeTransformation, ExcludeNodeTransformation() {
+    private val skipElementTypes = listOf("(", ")", "{", "}", "arrow", "semicolon", "comma", "dot")
```

Comment:
If you want to use `in` it might be a good idea to make a set rather than a list.